src/ORM/DataExtension.php

```diff
@@ -84,18 +114,37 @@ public function populateDefaults()
     {
     }
 
+
     public function can($member)
     {
     }
 
+    /**
+     * Whether or not the given $member is able to edit this decorated object.
+     *
+     * @param  Member $member
```

Comment:
We need to import the namespace for Member as well, up the top of the file: `use SilverStripe\Security\Member;`